test/utils/test_utils.js

```diff
@@ -19,10 +19,8 @@ testUtils.roundDecimal = function (item) {
     return Math.round( item * 100) / 100;
 };
 
-testUtils.testTidFromDate = function testTidFromDate(date, useCassTicks) {
-    var tidNode = new Buffer([0x01, 0x23, 0x45, 0x67, 0x89, 0xab]);
-    var tidClock = new Buffer([0x12, 0x34]);
-    return new TimeUuid(date, useCassTicks ? null : 0, tidNode, tidClock).toString();
+testUtils.testTidFromDate = function testTidFromDate(date) {
+    return new uuid({ msecs: date.getTime() }).toString();
```

Comment:
This is not entirely correct. As you can see, in the previous version we've used dummy but fixes clockSeq, node id, and dummy nanoseconds. This was done to make TID predictable based on the time and time only. Now it's not and some of the tests will fail.

Reply:
Added the values into the options object. In your previous wrapper you had a counter for ns. Is that used? Do I need to write a similar wrapper for uuid?